en/uavcan/README.md

```diff
@@ -1,49 +1,55 @@
-# UAVCAN Introduction
+# UAVCAN
 
-![UAVCAN Logo](../../assets/uavcan/uavcan_logo_transparent.png)
+<img style="float:right; width: 200px ; padding: 10px;" src="../../assets/uavcan/uavcan_logo_transparent.png" alt="UAVCAN Logo" /> [UAVCAN](http://uavcan.org) is an onboard network which allows the autopilot to connect to avionics/peripherals.
+In contrast to hobby-grade devices it uses rugged, differential signalling, supports firmware upgrades over the bus and supports status feedback from peripherals.
```

Comment:
This is a bit strong and inaccurate
- there's also hobby-grade uavcan devices
- there's industrial-grade devices that don't use uavcan (including fw update support and feedback)

Reply:
Thanks, I'll just remove "in contrast to ...".